include/cudaq/Optimizer/Transforms/Passes.h

```diff
@@ -30,6 +30,7 @@ void registerAggressiveEarlyInlining();
 
 void registerUnrollingPipeline();
 
+std::unique_ptr<mlir::Pass> createApplyControlNegationsPass();
```

Comment:
```suggestion
```
We can now use the "new MLIR way" for the pass creating functions. They get auto-generated now.